### PR TITLE
Balance Pass on Shotgun Expertise

### PR DESCRIPTION
**Pull request by Turtlelord26**
Shotguns also live in shorter-ranged spaces, but where SMGs spread their damage out among many bullets, Shotguns blast things in more discrete packages of death.
Strengths: point-blank damage spike, spreading vs solid shot versatility, generally high damage
Weaknesses: short-range, single-target, low APLs

0: Range bracket increase applied to R1. Keeping this number low for now because R1 buckshot is terrifyingly high damage.
12: Similarly to how SMGs were specialized by their expertise feat for the suppression role, I'm thinking I will specialize shotguns for room-clearing. To that goal, this bonus replaces Acrobatics for maneuvers while clearing a room, such as diving prone without losing aim on a target.
24: The attempt here is to support the versatility of different shot types by increasing the speed with which one can switch between them. I found no way to gamify '+reload when switching ammo types' so we have to go with flat +reload.
36: Negate hip-firing Accuracy penalty is technically a conditional accuracy buff. More conditional than most, but also pretty powerful and definitely reinforces the room-clearing role. Just subtly enough that people will feel warm and fuzzy for figuring that out, I hope.
48: R2 selected as it is the longest range spreading shot (e.g. buck or flechette) can be fired.
60: Keeping old capstone for now, it's good at alleviateing the APL issue. Will keep an eye on it though because it might be much weaker (or stronger, it's surprisingly hard to armchair evaluate) than the very similar capstone I gave pistols.

### CODE REVIEW

File: CharacterCreation/03_Feats.txt [hunk 1]
```diff
@@ -638,12 +638,20 @@ have no prerequisites.
...
-36:	-0.5 miss chance @ all ranges			
-48:	+10% perception checks for awareness within 25m.			
-60:	+10% Base damage. If you hit armor with a higher APL than your shotgun, you deal half damage to its AS instead of no damage (and full damage to health if you deplete the AS).
+0:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+12:	May use Weapons - Shotgun in place of Acrobatics for checks to execute 
+	a physical maneuver following violent entry of a room.
+24:	If you succeed a reload check by at least 20, reduce the time taken to 
+	reload by one Action, stacking with other reductions (such as from crits), 
+	to a minimum of one Action.
```
Comment (bleehu) [inline CharacterCreation/03_Feats.txt]: I really really like this bonus! Clearly you have a zen-like understanding of what makes shotguns different from the rest and what makes them fun! Nailed it!

File: CharacterCreation/03_Feats.txt [hunk 2]
```diff
@@ -638,12 +638,20 @@ have no prerequisites.
...
-0:	+10m to range R3	
-12:	-15% reload miss chance			
-24:	+5 Gun Reflex				
-36:	-0.5 miss chance @ all ranges			
-48:	+10% perception checks for awareness within 25m.			
-60:	+10% Base damage. If you hit armor with a higher APL than your shotgun, you deal half damage to its AS instead of no damage (and full damage to health if you deplete the AS).
+0:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+12:	May use Weapons - Shotgun in place of Acrobatics for checks to execute 
+	a physical maneuver following violent entry of a room.
```
Comment (bleehu) [inline CharacterCreation/03_Feats.txt]: I think we should maybe formally write out how room-clearing works. Your bonus here has some real potential, but the mechanics it's resting on aren't developed enough to support it.

Comment (bleehu) [reply CharacterCreation/03_Feats.txt]: For instance, the weapon's reflex save is supposed to be what counts in room clearing, not really Acrobatics.

File: CharacterCreation/03_Feats.txt [hunk 3]
```diff
@@ -638,12 +638,20 @@ have no prerequisites.
...
+24:	If you succeed a reload check by at least 20, reduce the time taken to 
+	reload by one Action, stacking with other reductions (such as from crits), 
+	to a minimum of one Action.
+36:	Negate the hip-firing Accuracy penalty against enemies within R1 while 
+	firing spreading shot.
```
Comment (bleehu) [inline CharacterCreation/03_Feats.txt]: oh hell ya!

**Review by bleehu — APPROVED** [review]
shipit! Great work!!!

File: CharacterCreation/03_Feats.txt (hunk 2 above)
Comment (Turtlelord26) [reply CharacterCreation/03_Feats.txt]: I was wary of weapon reflex at the time. I was specifically trying to describe a very active choice of Action to make it more skill-like and less reaction-like. I may not have succeeded in that, looking again.

Classifying as New Issue. Would you like to make it or shall I?

**Pull request merged**